schematools/types.py

```diff
@@ -7,17 +7,18 @@
 from string_utils import slugify
 
 import jsonschema
+from schematools import SCHEMA_VERSION
 
 SUPPORTED_REFS = {
     "https://geojson.org/schema/Geometry.json",
     "https://geojson.org/schema/Point.json",
     "https://geojson.org/schema/Polygon.json",
-    "https://schemas.data.amsterdam.nl/schema@v1.1.0#/definitions/id",
-    "https://schemas.data.amsterdam.nl/schema@v1.1.0#/definitions/class",
-    "https://schemas.data.amsterdam.nl/schema@v1.1.0#/definitions/dataset",
-    "https://schemas.data.amsterdam.nl/schema@v1.1.0#/definitions/year",
-    "https://schemas.data.amsterdam.nl/schema@v1.1.0#/definitions/uri",
-    "https://schemas.data.amsterdam.nl/schema@v1.1.0#/definitions/schema",
+    f"https://schemas.data.amsterdam.nl/schema@{SCHEMA_VERSION}#/definitions/id",
+    f"https://schemas.data.amsterdam.nl/schema@{SCHEMA_VERSION}#/definitions/class",
+    f"https://schemas.data.amsterdam.nl/schema@{SCHEMA_VERSION}#/definitions/dataset",
+    f"https://schemas.data.amsterdam.nl/schema@{SCHEMA_VERSION}#/definitions/year",
+    f"https://schemas.data.amsterdam.nl/schema@{SCHEMA_VERSION}#/definitions/uri",
+    f"https://schemas.data.amsterdam.nl/schema@{SCHEMA_VERSION}#/definitions/schema",
```

Comment:
Ik had de 1.1.0 en 1.1.1 versie laatst nodig om een ouder schema te parsen. Ik denk niet dat je deze hier zo dynamisch kan neerzetten

Reply:
Misschien het is dan beter om versie helemaal weg te gooien van deze REFs?

Reply:
@jjmurre @vdboor wat is je mening hier? Moeten we Schema Versie hardcoderen of niet?

Reply:
Ik zou voorlopig wel de schemaversie hier hardcoden.